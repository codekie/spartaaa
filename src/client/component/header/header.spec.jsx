import React from 'react';
import Header from './header.jsx';
import { shallow } from 'enzyme';
import TaskStatus from '../../../comm/task-status';

it('renders correctly', () => {
    expect(shallow(<Header connectionState={TaskStatus.pending} loading={false} refreshTasks={jest.fn()} />))
        .toMatchSnapshot();
});
