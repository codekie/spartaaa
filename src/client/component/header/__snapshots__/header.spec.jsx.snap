// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<PanelHeader
  className=""
  renderAs="div"
>
  <Level
    breakpoint={null}
    className=""
    renderAs="nav"
    style={Object {}}
  >
    <LevelSide
      align="left"
      className=""
      renderAs="div"
      style={Object {}}
    >
      <LevelItem
        className=""
        renderAs="div"
        style={Object {}}
      >
        <Image
          alt="32x32"
          className=""
          fallback="http//bulma.io/images/placeholders/480x480.png"
          size={32}
          src="test-file-stub"
          style={Object {}}
        />
      </LevelItem>
      <LevelItem
        className=""
        renderAs="div"
        style={Object {}}
      >
        This is Spartaaa!!!
      </LevelItem>
      <LevelItem
        className=""
        renderAs="div"
        style={Object {}}
      >
        <Button
          className=""
          color={null}
          disabled={false}
          fullwidth={false}
          inverted={false}
          isStatic={false}
          loading={false}
          onClick={[MockFunction]}
          outlined={false}
          remove={false}
          renderAs="button"
          reset={false}
          rounded={false}
          size={null}
          state={null}
          style={Object {}}
          submit={false}
          text={false}
        >
          Refresh
        </Button>
      </LevelItem>
    </LevelSide>
    <LevelSide
      align="right"
      className=""
      renderAs="div"
      style={Object {}}
    >
      <LevelItem
        className=""
        renderAs="div"
        style={Object {}}
      >
        <Loader
          className=""
          renderAs="div"
          style={Object {}}
        />
      </LevelItem>
      <LevelItem
        className=""
        renderAs="div"
        style={Object {}}
      >
        <ConnectionState
          connectionState="pending"
        />
      </LevelItem>
    </LevelSide>
  </Level>
</PanelHeader>
`;
